doc/aurqueue.md

```diff
@@ -0,0 +1,23 @@
+# aurqueue
+
+## Name
+
+aurqueue - Order dependencies using .SRCINFO files
+
+## Synopsis
+
+aurqueue pkgbase [pkgbase, ...]
+
+## Description
+
+aurqueue takes the names of packages and validates the dependencies using datamash. It then orders the dependency chains for the given packages.
```

Comment:
I'd focus on tsort, as datamash is an implementation detail (checking if a tsort argument is valid).